README.md

```diff
@@ -139,7 +139,46 @@ module.exports = app => {
   };
 };
 ```
+### Clinets Depend on Sentinels
 
+Before you start to use Redis Sentinel, please checkout the [document](https://redis.io/topics/sentinel) first.
+
+In controller, you also can use `app.redis` to get the redis instance Monitored by Redis Sentinel.
+
+```js
+
+// app/config/config.default.js
+
+exports.redis = {
+   client: {
+    sentinels: [
+      {
```

Comment:
egg-redis also supports clients configuration which should also considered.

Reply:
clients configuration doesn't concern about the type of client configuration, so i think clients configuration can still be supported.